dist/output.css

```diff
@@ -0,0 +1,535 @@
+/*
```

Comment:
this is a compiled folder that you don't want to commit. 

add `dist` to your `gitignore` and delete this file.